_pages/bajo.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Bajocode
```

Comment:
I believe the filename must be exactly identical to your github handle...i.e. bajocode.yaml.